### PR TITLE
fixed neo4j package issue

### PR DESCRIPTION
**Pull request by praveenkumarb1207**
The packages for neo4j spark connector in the maven are not working .  We have taken the shaded jars and published to azure artifacts and used in the neo4j.almaren code.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -14,18 +14,25 @@ scalacOptions ++= Seq("-deprecation", "-feature")
...
+  "com.modak.neo4j" %% "neo4j-connector-apache-spark" % "4.0.2_for_spark_2.4"
 )
 
 enablePlugins(GitVersioning)
 
-resolvers += ("Spark Packages Repo" at "http://dl.bintray.com/spark-packages/maven").withAllowInsecureProtocol(true)
 resolvers +=
   "Sonatype OSS Snapshots" at "https://oss.sonatype.org/service/local/repositories/releases/content"
 
+scalacOptions ++= Seq("-deprecation", "-feature")
```
Comment (mantovani) [inline build.sbt]: Why did you add it here?

**Review by mantovani — CHANGES_REQUESTED** [review]

File: build.sbt (hunk 1 above)
Comment (praveenkumarb1207) [reply build.sbt]: Removed it . It was copy pasted from other project. We don't need it here

File: build.sbt [hunk 2]
```diff
@@ -14,18 +14,23 @@ scalacOptions ++= Seq("-deprecation", "-feature")
...
 
 enablePlugins(GitVersioning)
 
-resolvers += ("Spark Packages Repo" at "http://dl.bintray.com/spark-packages/maven").withAllowInsecureProtocol(true)
 resolvers +=
   "Sonatype OSS Snapshots" at "https://oss.sonatype.org/service/local/repositories/releases/content"
 
+ThisBuild / useCoursier := false
+
+resolvers += "nabu-releases" at "https://pkgs.dev.azure.com/nabu-backend/Nabu-Bots/_packaging/nabu-releases/maven/v1"
```
Comment (mantovani) [inline build.sbt:30]: we don't need this.

**Review by mantovani — CHANGES_REQUESTED** [review]

File: build.sbt (hunk 2 above)
Comment (praveenkumarb1207) [reply build.sbt:30]: We need them because the packages for neo4j spark connector in the maven are not working . We have taken the shaded jars and published to azure artifacts and used in the neo4j.almaren code.

for this package , we need them 

“com.modak.neo4j" %% "neo4j-connector-apache-spark" % "4.0.2_for_spark_2.4"

**Comment by praveenkumarb1207** [comment]
Closing this pull request as the neo4j.alamren is public repo , we  cannot use modak azure artifcats inside build.sbt . We need to discuss on how to solve the package issue
